lib/stripe_mock/request_handlers/validators/param_validators.rb

```diff
@@ -65,71 +65,9 @@ def invalid_currency_message(my_val)
         "Invalid currency: #{my_val.downcase}. Stripe currently supports these currencies: #{SUPPORTED_CURRENCIES.join(", ")}"
       end
 
-      def validate_create_plan_params(params)
```

Comment:
Curious, do all the validations cause issues for us? Wonder if it's possible to identify the specific validations that cause issues and remove those ones specifically? I think it'll still be helpful to keep validations if we can to keep test behavior more similar to prod. What do you think?

Reply:
Yes, the price, product and currency check all failed. Had to remove it entirely. 

In the current version, it doesn't have these 2 validation methods. The plan is to fix the tests or the validations (e.g. the currency validation does not take symbols or lower cases). For now, I am removing them for unblocking ruby 3 upgrade, and will add a task in the backlog to revisit it.

Reply:
LOL gotcha, thank you for the additional context! Follow-up task sounds good to me 😄 I think we should probably consider upgrading our Stripe API version soon too as stripe-ruby-mock is now supporting a newer API version than the one we are using...